src/java/com/wolfssl/provider/jsse/WolfSSLEngineHelper.java

```diff
@@ -1369,6 +1374,51 @@ else if (peerAddr != null) {
         return ret;
     }
 
+    private void checkKeySize(WolfSSLSession ssl, boolean clientMode) throws SSLException, WolfSSLException {
+        int keySize = this.ssl.getKeySize();
+
+        // Before we update the cached values, and return from the handshake, we
```

Comment:
Please switch C++ style comments (`//`) to C-style to match our coding standards (`/* */`).